vars/registerDns.groovy

```diff
@@ -1,17 +1,22 @@
 import uk.gov.hmcts.contino.Consul
 import uk.gov.hmcts.contino.AppPipelineConfig
+import uk.gov.hmcts.contino.AzPrivateDns
 
 def call(Map params) {
   AppPipelineConfig config = params.appPipelineConfig
 
   withAksClient(params.subscription, params.environment) {
     Consul consul = new Consul(this, params.environment)
+    AzPrivateDns azPrivateDns = new AzPrivateDns(this, params.environment)
 
     // Staging DNS registration
     if (config.legacyDeploymentForEnv(params.environment)) {
       withIlbIp(params.subscription, params.environment) {
         consul.registerDns("${params.product}-${params.component}-${params.environment}-staging", env.TF_VAR_ilbIp)
+        azPrivateDns.registerAzDns("${params.product}-${params.component}-${params.environment}-staging", env.TF_VAR_ilbIp)
```

Comment:
is it worth try catching and ignoring failure for now? or do we want it to fail?